releasenotes/notes/improve-agent-start-restart-concurrency-642420923b9ea275.yaml

```diff
@@ -0,0 +1,13 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Improve agent starting and simultaneously restarting concurrency handling.
```

Comment:
This makes it sound like the change is about agent start/shutdown behavior, but it's really about the start/stop/restart agent commands. This seems like a better changelog for the follow PR you mention in the PR discription.

Reply:
thank you